addons/l10n_es/migrations/5.0/post-migrate_tags_on_taxes.py

```diff
@@ -0,0 +1,6 @@
+from openerp.modules.registry import RegistryManager
+
+def migrate(cr, version):
+    registry = RegistryManager.get(cr.dbname)
+    from openerp.addons.account.models.chart_template import migrate_tags_on_taxes
+    migrate_tags_on_taxes(cr, registry)
```

Comment:
If you have more than one localization module that calls this function, the same process will be repeated several times. 

Then the function

https://github.com/odoo/odoo/blob/4e11d2fcb0e3e6b830bbef82f1d0de383ef8abae/addons/account/models/chart_template.py#L24-L44

should include:

* One extra argument `module=None` for indicating a specific localization module.
* If set, then in this line 

  https://github.com/odoo/odoo/blob/4e11d2fcb0e3e6b830bbef82f1d0de383ef8abae/addons/account/models/chart_template.py#L34

  we should filter by that module
* and finally, pass `l10n_es` as argument here.

cc @qdp-odoo